src/types/zoneserver.ts

```diff
@@ -1,6 +1,7 @@
 export interface Client {
   isLoading: boolean;
   isMounted: boolean;
+  selectedVehicle: string;
```

Comment:
we should do that this field can be empty ( with ? )  and to delete it rather that setting it to "0"